biorbd_optim/penalty.py

```diff
@@ -169,6 +169,38 @@ def minimize_predicted_com_height(penalty_type, ocp, nlp, t, x, u, **extra_param
 
                 penalty_type._add_to_penalty(ocp, nlp, CoM_height, **extra_param)
 
+        @staticmethod
+        def minimize_contact_forces(penalty_type, ocp, nlp, t, x, u, contacts_idx=(), data_to_track=(), **extra_param):
+            from casadi import Function
+            from .dynamics import Dynamics
```

Comment:
Move this to the beginning of the file

Reply:
done

Reply:
At the beginning of the file (not of the function ;) )